# Description: Hack to make sure that the binding generator pulls in the
#  correct headers on trusty, where we're building with a version of GCC
#  that is different to the one that clang was built with. Temporary until
#  we figure out a better way to do this.
# Author: Chris Coulson <chris.coulson@canonical.com>
# Forwarded: not-needed

--- a/layout/style/ServoBindings.toml
+++ b/layout/style/ServoBindings.toml
@@ -1,6 +1,6 @@
 [build]
 args = [
-    "-x", "c++", "-std=c++14", "-fno-sized-deallocation",
+    "-x", "c++", "-std=c++14", "-fno-sized-deallocation", "-isystem", "/usr/lib/gcc-mozilla/include/c++/4.9",
     "-DTRACING=1", "-DIMPL_LIBXUL", "-DMOZ_STYLO_BINDINGS=1",
     "-DMOZILLA_INTERNAL_API", "-DRUST_BINDGEN", "-DMOZ_STYLO"
 ]
